src/practices/LanguageIndependent/DoesPullRequests.ts

```diff
@@ -0,0 +1,52 @@
+import { PracticeContext } from '../../contexts/practice/PracticeContext';
+import { sharedSubpath } from '../../detectors/utils';
+import { PracticeEvaluationResult, PracticeImpact } from '../../model';
+import { ReporterUtils } from '../../reporters/ReporterUtils';
+import { GitServiceUtils } from '../../services/git/GitServiceUtils';
+import { DxPractice } from '../DxPracticeDecorator';
+import { IPractice } from '../IPractice';
+import moment from 'moment';
+
+@DxPractice({
+  id: 'LanguageIndependent.DoesPullRequests',
+  name: 'Do PullRequests',
+  impact: PracticeImpact.medium,
+  suggestion: 'Do pull requests. It helps you catch the bad code before it is merged into the main codebase.',
+  reportOnlyOnce: true,
+  url: 'https://developerexperience.io/practices/pull-requests',
+})
+export class DoesPullRequests implements IPractice {
+  async isApplicable(): Promise<boolean> {
+    return true;
+  }
+
+  async evaluate(ctx: PracticeContext): Promise<PracticeEvaluationResult> {
+    if (ctx.fileInspector === undefined || ctx.collaborationInspector === undefined) {
+      return PracticeEvaluationResult.unknown;
+    }
+
+    const repoName = GitServiceUtils.getRepoName(ctx.projectComponent.repositoryPath, ctx.projectComponent.path);
+    const ownerAndRepoName = GitServiceUtils.getOwnerAndRepoName(repoName);
+
+    const pullRequests = await ctx.collaborationInspector.getPullRequests(ownerAndRepoName.owner, ownerAndRepoName.repoName);
+    const repoCommits = await ctx.collaborationInspector.getRepoCommits(ownerAndRepoName.owner, ownerAndRepoName.repoName);
+
+    let prDate;
+
+    if (pullRequests.items.length > 0) {
+      prDate = new Date(pullRequests.items[0].createdAt).getTime();
```

Comment:
`pullRequests.items[0]` - We can be sure that the first element is the newest? I'd prefer to use `sort` parameter to ensure the order of pull requests.